variables.tf

```diff
@@ -12,3 +12,9 @@ variable "policy" {
     description = "A valid IAM policy for the task"
     type        = "string"
 }
+
+variable "volume" {
+    description = "Volume block map with 'name' and 'host_path'. 'name': The name of the volume as is referenced in the sourceVolume. 'host_path' The path on the host container instance that is presented to the container."
```

Comment:
I'd probably shorten the description here a bit and refer to README for full guide on how to use this; just something to consider